src/codegen/metadata/attributes.py

```diff
@@ -0,0 +1,130 @@
+attributes = {
```

Comment:
This is just a sample metadata which contains the required data for generating some attributes present in `ai_channel.py`